README.md

```diff
@@ -44,6 +44,22 @@ uvicorn app.main:app --reload
 python -m pytest --cov-report term-missing --cov=app tests
 ```
 
+### Generating  personal JWT Secret Key
+'''shell
```

Comment:
Prefer supplying a one-liner they can run and it would just work. Something like:

```
python -c "
import secrets; from pathlib import Path;
f = Path('app/config.py');
f.write_text(f.read_text().replace('JWT_KEY_PLACEHOLDER', secrets.token_hex(32), 1));
"
```

Put this in the Linux & Windows instruction, and change in the config the line to `JWT_KEY = "JWT_KEY_PLACEHOLDER"`